build.gradle

```diff
@@ -60,7 +60,8 @@ ext {
             junitPlatformEngine : [group: 'org.junit.platform', name: 'junit-platform-engine', version: '1.8.2'],
             hamcrest            : [group: 'org.hamcrest', name: 'hamcrest-core', version: '1.3'],
             junit_dataprovider  : [group: 'com.tngtech.java', name: 'junit-dataprovider', version: '1.11.0'],
-            mockito             : [group: 'org.mockito', name: 'mockito-core', version: '4.4.0'],
+            mockito             : [group: 'org.mockito', name: 'mockito-core', version: '4.6.1'],
+            mockito_junit5      : [group: 'org.mockito', name: 'mockito-junit-jupiter', version: '4.6.1'],
```

Comment:
FYI: A version ≥`4.7.0` broke the build with
> Execution failed for task ':archunit:compileJdk9testJava'.
> java.time.DateTimeException: Invalid value for MonthOfYear (valid values 1 - 12): 0

(which sounds like [a JDK bug](https://bugs.openjdk.org/browse/JDK-8184940))